hazelcast-sql/src/main/java/com/hazelcast/sql/impl/calcite/parse/UnsupportedOperationVisitor.java

```diff
@@ -182,19 +170,26 @@
         SUPPORTED_OPERATORS.add(SqlStdOperatorTable.CURRENT_TIMESTAMP);
         SUPPORTED_OPERATORS.add(SqlStdOperatorTable.LOCALTIMESTAMP);
         SUPPORTED_OPERATORS.add(SqlStdOperatorTable.LOCALTIME);
-
-        // Other/Extensions
-        SUPPORTED_OPERATORS.add(SqlOption.OPERATOR);
     }
 
-    private final Set<SqlOperator> extensionOperators;
+    private final SqlValidatorCatalogReader catalogReader;
+
+    private final Set<SqlKind> extensionSqlKinds;
+    private final Set<SqlOperator> extensionSqlOperators;
+
+    private boolean exclusivelyImdgStatement;
 
     UnsupportedOperationVisitor(
             SqlValidatorCatalogReader catalogReader,
-            Set<SqlOperator> extensionOperators
+            Set<SqlKind> extensionSqlKinds,
+            Set<SqlOperator> extensionSqlOperators
     ) {
         this.catalogReader = catalogReader;
-        this.extensionOperators = extensionOperators;
+
+        this.extensionSqlKinds = extensionSqlKinds;
```

Comment:
The same concern as in the parser. It may happen that Jet would require a custom syntax somewhere deep inside some other fragment (e.g. SELECT statement), We will not be able to extend the `UnsupportedOperationVisitor` without a separate IMDG release.

Reply:
`UnsupportedOperationVisitor` is no longer shared